dll/win32/profmap/profmap_main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * PROJECT:         ReactOS User Profile Mapping API
```

Comment:
We have different format for the file header right now:
https://reactos.org/wiki/Coding_Style#File_Structure

Reply:
Fixed. Thanks.
I looked up a headers for some other source files and hope that `ReactOS system libraries` is correct.